corehq/ex-submodules/casexml/apps/phone/management/commands/migrate_synclogs_to_sql.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function
+from __future__ import absolute_import
+import datetime
+from django.core.management import BaseCommand
+from corehq.util.log import with_progress_bar
+from casexml.apps.phone.models import SyncLog, SyncLogSQL, properly_wrap_sync_log, \
+    synclog_to_sql_object
+from dimagi.utils.couch.database import iter_docs_with_retry
+
+
+class Command(BaseCommand):
```

Comment:
@sravfeyn are you planning on using this? If not you might as well remove it from the code.